content/applications/finance/fiscal_localizations/hong_kong.rst

```diff
@@ -0,0 +1,63 @@
+=========
+Hong Kong
+=========
+
+Configuration
+=============
+
+:ref:`Install <general/install>` the :guilabel:`Hong Kong - Accounting` module to
+get the features of the HK localization.
+
+.. image:: hong_kong/hk_01.png
+   :align: center
+
+Add FPS QR Codes to invoices
+============================
+
+Activate QR Code feature
+------------------------
+
+Go to :menuselection:`Accounting --> Configuration --> Settings`
+and activate the **QR Code** feature.
+
+.. image:: hong_kong/hk_02.png
+   :align: center
+
+Configure your FPS bank account
+-------------------------------
+
+Go to :menuselection:`Contacts --> Configuration --> Bank Accounts`,
+select a Account Holder with Hong Kong as country, open your FPS bank
+account and fill in the **FPS Type** and **FPS ID/Phone Number/Email
+Address** fields.
+
+.. image:: hong_kong/hk_03.png
+   :align: center
+
+Configure your Bank Account's journal
+-------------------------------------
+
+Go to :menuselection:`Accounting --> Configuration --> Journals`, open your
+bank journal, then fill out the Account Number and Bank under the Journal
+Entries tab.
+
+.. image:: hong_kong/hk_04.png
+   :align: center
```

Comment:
Suggestion to only add screenshots of the relevant area, rather than keeping wide areas with arrows or squares to show what we want to illustrate.